### PR TITLE
feat: provide uds-releaser option for publishing uds packages

### PR DESCRIPTION
**Pull request by corang**
## Description

...

## Checklist before merging

- [ ] ADR proposed if making an architectural change to the repo
- [ ] Tests run, docs added or updated as needed

### CODE REVIEW

File: .github/workflows/release-uds.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
```
Comment (Racer159) [inline .github/workflows/release-uds.yaml:2]: We should just cutover the existing release workflow.

File: tasks.yaml [hunk 2]
```diff
@@ -7,6 +7,7 @@ includes:
   - deploy: ./tasks/deploy.yaml
   - lint: ./tasks/lint.yaml
   - publish: ./tasks/publish.yaml
+  - release: ./tasks/release.yaml
```
Comment (Racer159) [inline tasks.yaml:10]:
```suggestion
```

**Review by Racer159 — CHANGES_REQUESTED** [review]

File: releaser.yaml [hunk 3]
```diff
@@ -3,14 +3,8 @@
 
 flavors:
   - name: upstream
-    # x-release-please-start-version
-    version: 1.4.0
-    # x-release-please-end
+    version: 1.27.3
```
Comment (Racer159) [inline releaser.yaml:6]: is it possible to renovate these with a `# renovate` comment?

**Comment by corang** [comment]
Not super confident in those renovate comments but I don't know how to see if they're working

**Comment by Racer159** [comment]
> Not super confident in those renovate comments but I don't know how to see if they're working

You can easily test in a fork:

https://github.com/Racer159/uds-common/commit/5af5a0a1c6f71374941a18b70cd680b16234e5c5

You just have to enable renovate for your personal repo and drop the host rules:

https://github.com/Racer159/uds-common/commit/cdb40a521d8e33b8b9f550674ba13b9813e17474#diff-7ad11411d2424c8b7004545af4a91293b3a953c25e697b262f27a57bdba2b414L31

And point the renovate.json to the one in your repo:

https://github.com/Racer159/uds-common/commit/6bbae79f8c7b1c267d0b29f315d9fb33938a5dcf

Then enable issues in settings and look through the dashboard: https://github.com/Racer159/uds-common/issues/3 or go through PRs: https://github.com/Racer159/uds-common/pull/13

File: bundle/uds-bundle.yaml [hunk 4]
```diff
@@ -5,13 +5,9 @@ kind: UDSBundle
 metadata:
   name: test
   description: The testing UDS bundle
-  # x-release-please-start-version
-  version: 1.4.0
-  # x-release-please-end
+  version: 1.27.3
```
Comment (Racer159) [inline bundle/uds-bundle.yaml:8]:
```suggestion
  version: dev
```

File: bundle/uds-bundle.yaml [hunk 5]
```diff
@@ -5,13 +5,9 @@ kind: UDSBundle
...
 
 packages:
   - name: nginx
     path: ../
-    # x-release-please-start-version
-    ref: 1.4.0
-    # x-release-please-end
+    ref: 1.27.3
```
Comment (Racer159) [inline bundle/uds-bundle.yaml:13]:
```suggestion
    ref: dev
```

File: zarf.yaml [hunk 6]
```diff
@@ -6,9 +6,7 @@ kind: ZarfPackageConfig
 metadata:
   name: nginx
   description: UDS nginx package
-  # x-release-please-start-version
-  version: 1.4.0
-  # x-release-please-end
+  version: 1.27.3
```
Comment (Racer159) [inline zarf.yaml:9]:
```suggestion
  version: dev
```

**Review by Racer159 — CHANGES_REQUESTED** [review]

**Review by Racer159 — APPROVED** [review]

**Pull request merged**